src/Detector.php

```diff
@@ -64,6 +64,21 @@ public function determineMimeType(\SplFileObject $file): string
         return 'text/plain; charset=utf-8';
     }
 
+    /**
+     * Read width amd height from a image file
+     *
+     * @param string $filePath
+     */
+    public function determineDimensions(string $filePath): array
```

Comment:
I really don't want to limit this to file paths on disk, as it means anyone working with a stream needs to potentially write some, or all, of the stream out to disk to use this function. Additionally, in it's current form, it's little more than a wrapper around `getimagesize`, but we can come up with a long-term solution for evolving this function:

Would you change this to accept an `SplFileInfo` and then change the `file_exists` check below to be `$stream->isReadable()`?

Anyone working with files on disk should be able to construct an `SplFileInfo` like so:

```php
$stream = new \SplFileInfo("image.jpg");
$detector->determineDimensions($stream);
```

Please don't forget to update the unit tests!